cli/src/semgrep/commands/ci.py

```diff
@@ -616,7 +616,7 @@ def ci(
             )
 
         logger.info(
-            f"    https://semgrep.dev/orgs/{scan_handler.deployment_name}/findings"
+            f"    https://semgrep.dev/orgs/{scan_handler.deployment_name}/findings?&repo={metadata.repo_display_name}"
```

Comment:
Did you include the ref filter and then revert it? The `?&` here looks off, and I just saw that it is actually in the PR description

Reply:
oh no the merge must have done something weird

Reply:
Fixed